### PR TITLE
Split the v0.2.0 report data into indidvidual report files

### PR DESCRIPTION
**Pull request by Munsio**

### CODE REVIEW

**Comment by Munsio** [comment]
@zimmski please review. I've split up all the logs and sampled a few of the outputs if they are correct.

File: report/log-splitter.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func main() {
```
Comment (zimmski) [inline report/log-splitter.go:11]: Let's drop that script. The next evalation results should split right away. Automate all the things!

File: report/v0.2.0/openrouter_google_palm-2-chat-bison-32k@golang@@golang-plain [hunk 2]
```diff
@@ -0,0 +1,58 @@
+2024/04/03 22:54:19 Evaluating model "openrouter/google/palm-2-chat-bison-32k" using language "golang" and repository "/home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/testdata/golang/plain"
```
Comment (zimmski) [inline report/v0.2.0/openrouter_google_palm-2-chat-bison-32k@golang@@golang-plain:1]: Now that i see this. Can we put the files in directories instead: So `report/v0.2.0/openrouter_google_palm-2-chat-bison-32k@golang@@golang-plain` becomes

`$path-for-reports/$version/$model/$language/$repository.log`

Sorry for the change.

File: report/v0.2.0/evaluation.csv [hunk 3]
```diff
@@ -0,0 +1,125 @@
+model,files-total,files-executed,files-problems,coverage-statement
```
[the remark this answers is not in the document]
Comment (Munsio) [reply report/v0.2.0/evaluation.csv:1]: I will use `/docs/reports`

File: report/v0.2.0/symflower_symbolic-execution@golang@@golang-plain [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+=== RUN   TestSymflowerPlain/#00
+--- PASS: TestSymflowerPlain (0.00s)
+    --- PASS: TestSymflowerPlain/#00 (0.00s)
+PASS
+coverage: 100.0% of statements
+ok      plain   0.001s  coverage: 100.0% of statements
+
+DONE 2 tests in 0.151s
+2024/04/03 23:07:15 Evaluated model "symflower/symbolic-execution" using language "golang" and repository "/home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/testdata/golang/plain": encountered 0 problems
+2024/04/03 23:07:15 Evaluating models and languages
```
Comment (zimmski) [inline report/v0.2.0/symflower_symbolic-execution@golang@@golang-plain:31]: That line needs to go in the evaluation.log

Comment (zimmski) [reply report/v0.2.0/symflower_symbolic-execution@golang@@golang-plain:31]: Also, missing the evaluation.log which should contain all remaining logs (so minus the logs of the models)

**Comment by Munsio** [comment]
@zimmski please check again if the logs are what you want. If it is okay I will remove the script and push again.

**Comment by zimmski** [comment]
> @zimmski please check again if the logs are what you want. If it is okay I will remove the script and push again.

I hate GitHub's PR functionality. You cannot comment with a thread unless you are in a file commenting there, you cannot review one commit after another because the "file viewed" functionality only exists in the "changes tab" where you see everything and and and... not loving this.

File: docs/reports/old.log [hunk 5]
```diff
@@ -0,0 +1,7218 @@
+$bash run.sh
```
Comment (zimmski) [inline docs/reports/old.log:1]: Please remove that file as well.

File: docs/reports/v0.2.0/evaluation.log [hunk 6]
```diff
@@ -0,0 +1,1793 @@
...
+plain.go]
+2024/04/03 23:07:14 Evaluating model "symflower/symbolic-execution" using language "golang" and repository "/home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/testdata/golang/plain"
```
Comment (zimmski) [inline docs/reports/v0.2.0/evaluation.log:1666]: @Munsio can you also add to your task list to change the repository path to be relative not absolute in the logs. The information should be as stable as possible for everyone. No matter where you run the evalation.

**Pull request merged**